client/src/pages/ClientDetailsPage.js

```diff
@@ -129,24 +132,28 @@ function ClientDetailsPage(props) {
       ) : (
         ""
       )}
+
       {!isInitialized ? (
-        <Container className='pt-5 d-flex justify-content-center'>
-          <Spinner variant='dark' animation='border' />
+        <Container className="pt-5 d-flex justify-content-center">
+          <Spinner variant="dark" animation="border" />
         </Container>
       ) : (
+        <Row>
+          <h1 className="title">Client Details</h1>
+        </Row>
+      )}
+
+      {isInitialized && authContext.currentUser.role === UserRoles.EMPLOYEE ? (

```

Comment:
Instead of using this ternary you can do:

`{isInitialized && content}`

Where content is:
```
<Row>title</Row>
<Row>
<Col>details</Col>
{authContext.currentUser.role === UserRoles.EMPLOYEE && <Col>topupform</Col>}
</Row>
{authContext.currentUser.role === UserRoles.EMPLOYEE && <Row>createneworder</Row>>}
<Row>orders</Row>
```